bedrock/settings/__init__.py

```diff
@@ -179,10 +104,136 @@
     DEV_LANGUAGES = PROD_LANGUAGES
     LANGUAGE_CODE = "en"  # Pocket uses `en` not `en-US`
 
+    # CSP settings for POCKET, expanded upon later:
+    _csp_default_src = [
+        "'self'",
+        "*.getpocket.com",
+    ]
+    _csp_img_src = [
+        "data:",
+        "www.googletagmanager.com",
+        "www.google-analytics.com",
+    ]
+    _csp_script_src = [
+        # TODO fix use of OptanonWrapper() so that we don't need this
+        "'unsafe-inline'",
+        # TODO onetrust cookie consent breaks
+        # blocked without unsafe-eval. Find a way to remove that.
+        "'unsafe-eval'",
+        "www.googletagmanager.com",
+        "www.google-analytics.com",
+        "tagmanager.google.com",
+        "cdn.cookielaw.org",
+        "assets.getpocket.com",  # allow Pocket Snowplow analytics
+    ]
+    _csp_style_src = [
+        "'unsafe-inline'",
+    ]
+    _csp_child_src = []
+    _csp_connect_src = [
+        "www.googletagmanager.com",
+        "www.google-analytics.com",
+        "o1069899.sentry.io",
+        "o1069899.ingest.sentry.io",
```

Comment:
Are we using Sentry for Pocket? Happy to leave this in if that is the plan, but if not we can probably remove these?